### ISSUE TITLE
Connection RDP template wont add all options only some but not allow font smoothing:i:1 i cant get it turned on

### ISSUE THREAD

**Issue by pbvdven** [bolkedebruin/rdpgw#78]
i tried to set some options but they wont all come true but the one big problem allow font smoothing:i:1 i tried everything to get it turned ON but it wont turn on.

**spit out by the gateway**

gatewayhostname:s:del
full address:s:DHL-VDI-WIN11:3389
username:s:pbvdven
gatewaycredentialssource:i:5
gatewayprofileusagemethod:i:1
gatewayusagemethod:i:1
gatewayaccesstoken:s: del
enablecredsspsupport:i:0
networkautodetect:i:0
connection type:i:6
audiocapturemode:i:1
camerastoredirect:s:*
drivestoredirect:s:*

**Connection settings of the RDP file** 
i would like all settings turned on
![image](https://github.com/bolkedebruin/rdpgw/assets/39600990/cbd4b4c9-2293-4837-8afa-dc3c4fd630f1)

**Default rdp template**

enablecredsspsupport:i:0
audiocapturemode:i:1
videoplaybackmode:i:1
connection type:i:6
networkautodetect:i:0
bandwidthautodetect:i:1
audiomode:i:0
redirectprinters:i:1
redirectlocation:i:0
redirectcomports:i:0
redirectsmartcards:i:1
redirectwebauthn:i:1
redirectclipboard:i:1
redirectposdevices:i:0
camerastoredirect:s:*
**allow font smoothing:i:1
allow desktop composition:i:1
disable full window drag:i:0
disable menu anims:i:0
disable themes:i:0
disable cursor setting:i:0
bitmapcachepersistenable:i:1**
drivestoredirect:s:*
autoreconnection enabled:i:1

Maybe im doing something working or i misunderstood how it works. i want a RDP file with storage, camara, mic, sound, and  all performance options turned on. i also need enablecredsspsupport:i:0 because of the custom auth provider for windows all works now only not the performance settings and the front is very bad now.

**Comment by christiaangoossens** [comment]
@pbvdven This is still a bug, but setting the connection mode to autodetect (`
connection type:i:7`) does fix the issue, al be it only when it auto detects bandwidth to use everything.

**Comment by pbvdven** [comment]
> @pbvdven This is still a bug, but setting the connection mode to autodetect (` connection type:i:7`) does fix the issue, al be it only when it auto detects bandwidth to use everything.

Yes setting to 7 auto detect that is what i do at the moment it works but would be nicer to set actual options. But i still didn’t find out why some options wont get passed on to the output rdp file and some actually do end up in the rdp file. Thanks for your reply.

**Comment by bolkedebruin** [comment]
This should be fixed in the current master,
